1.Fuzzing_technology/README.md

```diff
@@ -304,11 +342,11 @@ D) Generating mutations with Radamsa and observing program execution
 ---
 
 ### A) Create a mutator
-In the context of fuzzing, mutation refers to the process of generating new test inputs by modifying existing inputs in order to explore different program behaviors and potentially trigger software vulnerabilities or bugs. 
+In the context of fuzzing, mutation refers to the process of generating new test inputs by modifying existing inputs in order to explore different program behaviors and potentially trigger software vulnerabilities or bugs.
 
 Design a mutator that takes a valid URL as input and creates its mutations.
 Sample seed input that could be used: http://www.google.com/search?q=fuzzing
-Generate 20 mutated cases. 
+Generate 20 mutated cases.
```

Comment:
Instead of providing number for mutated cases, we should maybe use tie. Generate samples over the time of 2-4 minutes and use them. It will give us much more data for testing. Otherwise students will do bare minimum. 
It should be clear that fuzzing is never finished. We will do it as much as we have resources, so small samples might give students the wrong idea.